beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -1803,26 +1803,30 @@ pub static KZG_VERIFICATION_BATCH_TIMES: LazyLock<Result<Histogram>> = LazyLock:
         "Runtime of batched kzg verification",
     )
 });
+/// For reference on how the kzg data column verification buckets were set, here are some numbers for 48 blobs:
+/// * 1 column batch: 5.76 ms
+/// * 8 columns batch: 34.3 ms
+/// * 64 columns batch: 257 ms
+/// * 128 columns batch: 508 ms
 pub static KZG_VERIFICATION_DATA_COLUMN_SINGLE_TIMES: LazyLock<Result<Histogram>> =
-    LazyLock::new(|| {
-        try_create_histogram_with_buckets(
-            "beacon_kzg_verification_data_column_single_seconds",
-            "Runtime of single data column kzg verification",
-            Ok(vec![
-                0.0005, 0.001, 0.0015, 0.002, 0.003, 0.004, 0.005, 0.007, 0.01, 0.02, 0.05,
-            ]),
-        )
-    });
+    // 5 exponential buckets between 0.002 and 0.032 seconds, with more granularity on the lower end.
```

Comment:
Isn't 32ms quite small for the largest bucket? Average should be around 5ms I know, but maybe we could push out a bit higher in case of slow hw (maybe 128ms?)

Reply:
yeah agree - my thinking was if it's above 32ms, then something is seriously wrong and the node will likely suffer, but adding 2 extra buckets sounds reasonable to me.